src/main/java/me/imoltres/bbu/game/Game.kt

```diff
@@ -180,8 +181,10 @@ class Game {
         end = Bukkit.getWorld(NamespacedKey.minecraft("the_end"))!!
         //Get spawn world
         spawnWorld = WorldCreator("bbuSpawnWorld").generator(EmptyChunkGenerator()).createWorld()!!
+        //Get death match world
+        deathMatchWorld = WorldCreator("bbuDeathMatchWorld").generator(EmptyChunkGenerator()).createWorld()!!
```

Comment:
Unnecessary, but can add configuration to make this available

Reply:
Completely removed